### ISSUE TITLE
Add Flower Baseline: FLANDERS

### ISSUE THREAD

**Issue by edogab33** [adap/flower#2595]
### Paper

Gabriele Tolomei, Edoardo Gabrielli, Dimitri Belli, Vittorio Miori. (2023). A Byzantine- Resilient Aggregation Scheme for Federated Learning via Matrix Autoregression on Client Updates.

### Link

https://arxiv.org/abs/2303.16668

### Maybe give motivations about why the paper should be implemented as a baseline.

I'm one of the authors. To the best of my knowledge, this is the first work to offer a new approach to the problem of model poisoning attacks by analyzing the history of model updates sent by the clients through an anomaly detection algorithm.

Our results show that FLANDERS can mitigate extremely severe attacks (i.e. where up to 95% of clients are byzantine).

### For first time contributors

- [x]  Read the [`first contribution` doc](https://flower.dev/docs/first-time-contributors.html)
- [x]  Complete the Flower tutorial
- [x]  Read the Flower Baselines docs to get an overview:
    - [x]  [https://flower.dev/docs/using-baselines.html](https://flower.dev/docs/using-baselines.html)
    - [x]  [https://flower.dev/docs/contributing-baselines.html](https://flower.dev/docs/contributing-baselines.html)

### Prepare - understand the scope

- [X] Read the paper linked above
- [x] Create the directory structure in Flower Baselines (just the `__init__.py` files and a `README.md`)
- [x] Before starting to write code, write down all of the specs of this experiment in a README (dataset, partitioning, model, number of clients, all hyperparameters, …)
- [x] Open a draft PR

### Implement - make it work

- [x] Implement some form of dataset loading and partitioning in a separate `dataset.py` (doesn’t have to match the paper exactly)
- [x] Implement the model in PyTorch
- [x] Write a test that shows that the model has the number of parameters mentioned in the paper
- [x] Implement the federated learning setup outlined in the paper, maybe starting with fewer clients
- [x] Plot accuracy and loss
- [x] Run it and check if the model starts to converge

### Align - make it converge

- [x] Implement the exact data partitioning outlined in the paper
- [x] Use the exact hyperparameters outlined in the paper
- [x] Make it converge to roughly the same accuracy that the paper states
- [x] Commit the final hyperparameters and plots
- [ ] Mark the PR as ready

**Comment by jafermarq** [comment]
Hi @edogab33 , thanks for opening this issue and propose contributing `FLANDERS`. Please follow the guide in `/baselines` to start with your baseline: https://github.com/adap/flower/tree/main/baselines. There you'll find linked a more detailed step-by-step guide if you need it. You can always refer to the baselines already in the [new format](https://flower.dev/docs/baselines/index.html) as a reference. 

Sorry if there was some confusion, the template assigned to `baseline` issues is yet to be updated to reflect the new way of contributing baselines.

**Comment by jafermarq** [comment]
Hi @edogab33 , the baseline is merged! Having a baselines with comprehensive experiments like yours is going to make it much easier for others to start working in the areas of model poisoning + FL. Amazing!

**Comment by edogab33** [comment]
Happy to see my work in Flower :D

Thanks for your efforts and suggestions @jafermarq!
